Bug 1127955 - converted text to aria-label

jobara opened the pull request:
Converted the past-other-month and future-other-month to be aria-labels instead of text nodes.

https://bugzilla.mozilla.org/show_bug.cgi?id=1127955

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/556024?v=3" alt="jobara" width="50px" height="50px" /> Justin Obara (jobara)](https://github.com/jobara) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b344cfd6e5b3)

yzen commented on apps/calendar/locales/calendar.en-US.properties:
```diff
@@ -269,8 +269,8 @@ busy[few] = {{n}} events
 present.ariaLabel = Today
-past-other-month = Other month
-future-other-month = Other month
+past-other-month.ariaLabel = Other month
```
Here and below: need to rename the key to something new (l10n requirement for updating strings).

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/556024?v=3" alt="jobara" width="50px" height="50px" /> Justin Obara (jobara)](https://github.com/jobara) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c4e6721c3c99)

jobara committed:
Bug 1127955 - converted text to aria-label

Converted the past-other-month and future-other-month to be aria-labels instead of text nodes.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/556024?v=3" alt="jobara" width="50px" height="50px" /> Justin Obara (jobara)](https://github.com/jobara) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=26876f02cbc9)
